tasks/configure.yml

```diff
@@ -52,7 +52,7 @@
 
 # Using sudo because become_user may be overridden
 - name: Initialize database
-  command: sudo -Hu {{ postgres_user }} initdb --locale {{ postgres_locale }} -E {{ postgres_encoding }} -D {{ postgres_data_directory }}
+  command: sudo -Hu {{ postgres_user }} {{ postgres_initdb | default('initdb') }} --locale {{ postgres_locale }} -E {{ postgres_encoding }} -D {{ postgres_data_directory }}
```

Comment:
why not use the `defaults` folder?

Reply:
Why would I?

Reply:
Because all default values would then reside in the same place. You even recommend to look vars up there in the readme ;)